README.md

```diff
@@ -1 +1,113 @@
-Hens never remembers to create a README.
+# gpii-launcher
+
+This package provides a `gpii.launcher` [Fluid component](http://docs.fluidproject.org/infusion/development/UnderstandingInfusionComponents.html)
+that standardizes the launching of fluid components with custom options.  The component is based on [yargs](http://yargs.js.org/) and [Kettle](https://github.com/fluid-project/kettle).
+
+# `gpii.launcher`
+
+The launcher component uses yargs to generate a set of merged options that reflect (in order of precedence):
+
+1. Any options passed on the command line.
```

Comment:
Moving to "native" resolution using IoC within the files as suggested below would mean that users i) didn't have to learn ii) wouldn't have to be restricted by these funny precedence rules